.github/workflows/ISSUE_TEMPLATE/config.yml

```diff
@@ -1,14 +1,23 @@
 blank_issues_enabled: false
 contact_links:
-  - name: New Idea?
-    url: https://github.com/CommunityToolkit/Labs-Windows/discussions?discussions_q=category%3AExperiments+category%3AIdeas+category%3AProposals
-    about: Labs is where we propose and experiment with new ideas. Search ongoing experiments or contribute your own!
   - name: Questions about the Toolkit?
     url: https://github.com/CommunityToolkit/Windows/discussions/categories/q-a
     about: I have a question about how to use something in the toolkit.
-  - name: MVVM Toolkit, High Performance, or Diagnostic Package? 
+  - name: New Idea?
+    url: https://github.com/CommunityToolkit/Labs-Windows/discussions?discussions_q=category%3AExperiments+category%3AIdeas+category%3AProposals
+    about: Labs is where we propose and experiment with new ideas. Search ongoing experiments or contribute your own!
+  - name: WindowsAppSDK
+    url: https://aka.ms/windowsappsdk
+    about: "A set of new developer components and tools that represent the next evolution in the Windows app development platform"
```

Comment:
Let's put this down above Uno, we probably want to put something like you have for the others. "This repository is for the Windows Community Toolkit, for issues in the platform itself, see the WindowsAppSDK repo:"

Similarly, we probably want a "See the Uno Platform repo for issues specific to running on non-Windows platforms:"?

Reply:
Fixed the about fields, not sure what you mean by "put this down above Uno". The uno field is second to the bottom, with Discord last.

Reply:
Just meant it should be right before Uno instead of above the other Community Toolkits.